frontend/src/components/doctors/doctors/DoctorSmallCard.tsx

```diff
@@ -1,6 +1,13 @@
-import { Doctor, getDoctorMinimalDistance } from "../../../types/doctors/doctor";
+import {
```

Comment:
Please do some minimal responsiveness tests. Using chrome devtools or even without, after I changed the window size, I got some "pretty" results, such as:
![image](https://github.com/TheYellowHub/TheMap/assets/42554587/48017419-d291-4aed-9168-7fd939e76971)